### PR TITLE
Add discord button to top

### PR DESCRIPTION
**Pull request by Arokha**
dreamseeker_36fF2wTHKz

Also removes changelog button because we never use that changelog and I don't want to mislead people. They can use the link in the MOTD. Or we can make a link to the release branch PRs.

### CODE REVIEW

File: interface/skin.dmf [hunk 1]
```diff
@@ -1189,8 +1189,8 @@ window "mainwindow"
 		type = MAIN
 		pos = 281,0
 		size = 640x440
-		anchor1 = none
-		anchor2 = none
```
Comment (Arokha) [inline interface/skin.dmf:1193]: All this stuff appears to just be me opening the skin.dmf in 514 and it *having its way* with it.

**Comment by TheFurryFeline** [comment]
MOTD does not show on oldchat, though. :V

**Comment by Very-Soft** [comment]
> MOTD does not show on oldchat, though. :V ~At least that's been my experience for a while but outdated for at least a month or so, so maybe fixed by now. Doubt it, though.~

It shows on old chat if you ghost and respawn. Either way, link at the top now, better than in MOTD

**Comment by TheFurryFeline** [comment]
> > MOTD does not show on oldchat, though. :V ~At least that's been my experience for a while but outdated for at least a month or so, so maybe fixed by now. Doubt it, though.~
> 
> It shows on old chat if you ghost and respawn. Either way, link at the top now, better than in MOTD

My issue has been that it doesn't shown when you log in, but yeah, yay for link.

**Comment by TheFurryFeline** [comment]
Also, just noticing the branch name. Disco is ded ™️ 😛

**Pull request merged**